packages/create-pantheon-decoupled-kit/src/index.ts

```diff
@@ -85,6 +91,14 @@ export const main = async (
 		return;
 	});
 
+	function isDrupalCms(value: string): value is DrupalCMS {
+		return ['drupal', 'd9', 'd10'].includes(value);
+	}
+
+	function isWpCms(value: string): value is WordpressCMS {
+		return ['wordpress', 'wp'].includes(value);
+	}
```

Comment:
I like this. Can you move these funcs to `types` and make them arrow functions? I could see these being used elsewhere in the future and they feel like they would fit nicely with `isString`